### DISCUSSION TITLE
about/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#4] [Announcements]
# about/

https://blog.wohin.me/about/

**Comment by brant-ruan** [comment]
2023, keep hacking :-)

**Comment by AndrewZheng-AI** [comment]
thk for sharing :-)

**Reply by brant-ruan** [reply]
Thanks for your replying!

**Comment by brant-ruan** [comment]
2024, fight for our future and dreams, with things under control.

**Comment by zzhaire** [comment]
2024 ， 同济cs学弟赶工os课设时看到， thx for sharing :-)

**Reply by brant-ruan** [reply]
加油！

**Comment by zayarht3t** [comment]
hi, I learned from your kernel exploitation post and run the exploit, i get uid=0 root but when spawing shell system("/bin/sh") . seg fault

**Reply by zayarht3t** [reply]
sorry , I change os to parrot and exploit worked. In my kali, doesn't work

**Comment by cdxiaodong** [comment]
学习大佬

**Comment by 212121hanghang** [comment]
作者写的0day的笔记让我看得受益匪浅，而且笔记是我见过写过非常完美的了，搞懂了很多不理解的知识，不知道作者有没有兴趣写一下漏洞战争的漏洞分析笔记

**Reply by brant-ruan** [reply]
谢谢，暂时应该不会写了，但是写下来确实是一个很锻炼的过程。大家都可以写一些这样的笔记。

**Comment by bcdlbgm** [comment]
感谢你的博客

**Comment by lieInYourCloset** [comment]
great blog!

**Comment by yingzui** [comment]
想问一下大佬是如何优雅的用hugo编写博客并且同步的呢？

**Reply by brant-ruan** [reply]
我是用Typora写作，然后用hugo生成，传到服务器上，挂在nginx里。

**Comment by brant-ruan** [comment]
2025, keep calm and carry on :P

**Comment by CX330Blake** [comment]
Happy hacking! 想問一下 profile.wohin.me 這個站是用的什麼框架和主題 :-)

**Reply by brant-ruan** [reply]
你好，这个是一个静态HTML+CSS

**Comment by CX330Blake** [comment]
原來，謝謝回覆！

Bonan Ruan ***@***.***> 於 2025年11月5日 週三 下午1:25寫道：

**Comment by breayhing** [comment]
2025，keep hacking

**Comment by brant-ruan** [comment]
2026, keep healthy, brave, and grateful :-)

**Comment by wumiao233** [comment]
Thx for sharing!! I've learned a lot from blogs!❤
